First/src/firstPackage/Example1.java

```diff
@@ -0,0 +1,38 @@
+package firstPackage;
+
+public class Example1 {
+
+	public static boolean sameOccurrence(String s) {
+        // Create an array to count the occurrences of each character
+        int[] count = new int[26];
+        for (int i = 0; i < s.length(); i++) {
+            char c = s.charAt(i);
+            count[c - 'a']++;
```

Comment:
Same here, change class name and variable name

Reply:
Done !! Added new files named as EqualOccurance.java